.github/workflows/CI_Integration_Push.yml

```
@@ -0,0 +1,31 @@
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v4
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x ./gradlew
+
+      - name: Code Style Check
+        run: ./gradlew spotlessCheck
+
+      - name: Build with Gradle Wrapper
+        run: ./gradlew test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)